#! /bin/sh /usr/share/dpatch/dpatch-run
## 20120517-r35434.patch by James Healy <jimmy@deefa.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: The test suites for ruby 1.9.3p125 and 1.9.3p194 fails to 
## DP: complete when built against openssl 1.0.1. The issue was
## DP: fixed on trunk in revision 35434, this backports the fix
## DP: into the debian package.
## DP: TODO: remove this patch when 1.9.3p194+1 is available
 
@DPATCH@
Index: ruby1.9.1/lib/drb/ssl.rb
===================================================================
--- ruby1.9.1.orig/lib/drb/ssl.rb	2012-05-17 15:28:27.000000000 +1000
+++ ruby1.9.1/lib/drb/ssl.rb	2012-05-17 15:28:27.000000000 +1000
@@ -54,7 +54,7 @@
           return
         end
 
-        rsa = OpenSSL::PKey::RSA.new(512){|p, n|
+        rsa = OpenSSL::PKey::RSA.new(1024){|p, n|
           next unless self[:verbose]
           case p
           when 0; $stderr.putc "."  # BN_generate_prime
